From 02e9d25e7836bc5f350bc81d78ed2b8dd457f9b2 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Mon, 27 Oct 2014 09:24:46 -0700
Subject: [PATCH 25/28] Ticket #47928 - Disable SSL v3, by default.

Description: commit c1ecd8b659a0b8f7d84f8157cb69810c85ee26e4
introdueced a coverity defect: 12783 - Logically dead code

This patch removes the dead code.

https://fedorahosted.org/389/ticket/47928

Reviewed by rmeggins@redhat.com (Thank you, Rich!!)

(cherry picked from commit 77989d3bb5471542db50d66de77cc40fe4500cbd)
(cherry picked from commit 29a41604ce48855bd6d96bbd83cbae870deaa8d7)
---
 ldap/servers/slapd/ssl.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/ldap/servers/slapd/ssl.c b/ldap/servers/slapd/ssl.c
index 26ef251..f81d1fb 100644
--- a/ldap/servers/slapd/ssl.c
+++ b/ldap/servers/slapd/ssl.c
@@ -1215,7 +1215,6 @@ slapd_ssl_init()
     PRErrorCode errorCode;
     char ** family_list;
     char *val = NULL;
-    int rv = 0;
     PK11SlotInfo *slot;
     Slapi_Entry *entry = NULL;
 
@@ -1334,10 +1333,6 @@ slapd_ssl_init()
      * already been executed. */
     _security_library_initialized = 1; 
 
-    if ( rv != 0 ) {
-        return rv;
-    }
-
     return 0;
 }
 
-- 
1.9.3

